### PR TITLE
Implement send transaction component & connect welcome back flow

### LINKED ISSUE #594: Handle wallet API errors
Implement error handlers for the wallet API errors in the JSON-RPC client that are being used:
- Welcome back flow
- Try data request
- First time usage

### LINKED ISSUE #690: Implement Send tab
Migrate the Send tab from legacy sheikah.

### LINKED ISSUE #691: Implement Receive tab
Implement the Receive tab from legacy sheikah

### LINKED ISSUE #708: Connect "welcome back" flow
This depends on #707

### PR DESCRIPTION
**Pull request by Tommytrg**
This PR contains:
 - Refactor of the api client. Now there a wrapper for the client that has the wallet methods in its API.
- Connect welcome back flow with the wallet ( close #708 )
- Show errors in the  encrypt wallet step (Close #594)
- Implement send tab (close #690 )
- Implement receive tab (close #691 )

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -46,31 +46,73 @@ export class ApiClient {
...
-// TODO(#594): Handle errors in a proper way
-function handleResponse (response) {
-  return response || { error: Error('Wallet error occurred') }
-}
+  constructor () {
+    this.client = new ApiClient()
+  }
+  // TODO(#594): Handle errors in a proper way
+  _handleResponse (response) {
+    return { result: response } || { error: response }
```
Comment (tmpolaczyk) [inline src/api.js]: This doesn't do what you think it does

File: src/components/InputNumber.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        return this.value
+      },
+      set(inputValue) {
+        this.$emit('input', inputValue)
+      }
+    }
+  },
+  methods: {
+    increase() {
+      this.$emit('input', parseInt(this.value) + 1)
```
Comment (tmpolaczyk) [inline src/components/InputNumber.vue]: Just to be safe

```suggestion
      this.$emit('input', parseInt(this.value, 10) + 1)
```

File: src/components/WelcomeBack/WalletList.vue [hunk 3]
```diff
@@ -4,14 +4,16 @@
         Select which wallet do you want to unlock
       </p>
       <ul class="list">
-        <li v-for="wallet in wallets" :key="wallet.id">
+        <div class="wallets">
+        <li v-for="wallet in wallets"  :key="wallet.id">
```
Comment (tmpolaczyk) [inline src/components/WelcomeBack/WalletList.vue]:
```suggestion
        <li v-for="wallet in wallets" :key="wallet.id">
```

File: src/components/InputNumber.vue (hunk 2 above)
Comment (Tommytrg) [reply src/components/InputNumber.vue]: Do you think that the expected behavior is to change the base of the introduced number? IMHO we should only accept numbers

Comment (tmpolaczyk) [reply src/components/InputNumber.vue]: Entering a monetary value in hexadecimal should be a hard error.

**Pull request merged**